### ISSUE TITLE
better-tailwindcss/no-unregistered-classes misses some DaisyUI classes

### ISSUE THREAD

**Issue by sazzer** [schoero/eslint-plugin-better-tailwindcss#287]
### Description

Firstly, I'm not 100% sure this is the right project for this issue. Apologies if not!

I've just started using this on a new project, and have hit an interesting issue. The rule `better-tailwindcss/no-unregistered-classes` generally works fine. It covers all of the standard Tailwind classes, and even - surprisingly - picked up classes from DaisyUI. However, occasionally there are DaisyUI classes that doesn't recognise and therefore flags up, even though they work fine.

For example, `dui-footer` and `dui-footer-horizontal` are handled correctly by this rule, but `dui-footer-center` isn't and is flagged up as an unknown class, even though it works correctly in the browser.

(Note, I've got DaisyUI configured to use the prefix `dui-` so I can more easily distinguish them from normal Tailwind classes)

### Flavor

TSX

#### Tailwind config:
```css
@import "tailwindcss";
@plugin "daisyui" {
  prefix: "dui-";
  themes:
    light --default,
    dark --prefersdark;
}
```

#### ESLint config:
```json
  // React
  {
    files: ["**/*.tsx", "**/*.jsx"],
    ignores,
    settings: {
      react: {
        version: "detect"
      },
      "better-tailwindcss": {
        entryPoint: "src/ui/tailwind.css"
      }
    },
    plugins: {
      "better-tailwindcss": eslintPluginBetterTailwindcss,
      i18next,
      "jsx-a11y": jsxA11y,
      "@tanstack/router": pluginRouter,
      react,
      "react-hooks": reactHooks,
      "react-refresh": reactRefresh
    },
    rules: {
      ...eslintPluginBetterTailwindcss.configs["recommended"].rules,
      ...i18next.configs.recommended.rules,
      ...jsxA11y.configs.recommended.rules,
      ...pluginRouter.configs.recommended.rules,
      ...react.configs.recommended.rules,
      ...react.configs["jsx-runtime"].rules,
      ...reactHooks.configs.recommended.rules,

      "react-refresh/only-export-components": "warn",
      "react/prop-types": "off"
    }
```

#### TSX in question:
```tsx
      <div
        className={`
          dui-footer-center dui-footer dui-footer-horizontal border-t
          border-base-300 bg-base-200 px-10 py-4 text-base-content
        `}
      >
        {t("footer.copyright", { year: new Date().getFullYear() })}
      </div>
```

### Expected Behavior

The `dui-footer-center` is a valid DaisyUI class and shouldn't cause the linter to fail.

### Actual Behavior

The `dui-footer-center` class is flagged up and causes the linter to fail.

### ESLint Log

```shell
-> % npx eslint src/ui/pages/components/footer/footer.tsx 
/*! 🌼 daisyUI 5.5.14 */
/*! 🌼 daisyUI 5.5.14 */
/*! 🌼 daisyUI 5.5.14 */
/*! 🌼 daisyUI 5.5.14 */
/*! 🌼 daisyUI 5.5.14 */

/Users/coxg/source/me/tbrts/webapp/src/ui/pages/components/footer/footer.tsx
  66:11  warning  Unregistered class detected: dui-footer-center  better-tailwindcss/no-unregistered-classes

✖ 1 problem (0 errors, 1 warning)
```

### ESLint Config

```typescript
// React
  {
    files: ["**/*.tsx", "**/*.jsx"],
    ignores,
    settings: {
      react: {
        version: "detect"
      },
      "better-tailwindcss": {
        entryPoint: "src/ui/tailwind.css"
      }
    },
    plugins: {
      "better-tailwindcss": eslintPluginBetterTailwindcss,
      i18next,
      "jsx-a11y": jsxA11y,
      "@tanstack/router": pluginRouter,
      react,
      "react-hooks": reactHooks,
      "react-refresh": reactRefresh
    },
    rules: {
      ...eslintPluginBetterTailwindcss.configs["recommended"].rules,
      ...i18next.configs.recommended.rules,
      ...jsxA11y.configs.recommended.rules,
      ...pluginRouter.configs.recommended.rules,
      ...react.configs.recommended.rules,
      ...react.configs["jsx-runtime"].rules,
      ...reactHooks.configs.recommended.rules,

      "react-refresh/only-export-components": "warn",
      "react/prop-types": "off"
    }
```

### Versions

- ESLint: 9.39.2
- Parser: 8.51.0
- Plugin: 3.8.0
- Node: 25.1.0
